src/index.html

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<html lang="en">
+	<head>
+		<meta charset="UTF-8" />
+		<meta http-equiv="X-UA-Compatible" content="IE=edge" />
+		<meta name="viewport" content="width=device-width, initial-scale=1.0" />
+		<link rel="icon" type="image/x-icon" href="./assets/images/favicon.ico" />
+		<title>To-Do List</title>
+	</head>
+	<body>
+		<section class="main">
+			<div class="container">
+				<div class="header">
```

Comment:
Here, it would be better to use:
- `section` instead of `div` to wrap the content of a section of your page:
  https://github.com/muneebulrehman/todo-review/blob/582e368b6867ed8f4e7683e7bd3ffe52f964ef53/src/index.html#L12
- `header` instead of `div` to wrap the content of the header of a section of your page:
  https://github.com/muneebulrehman/todo-review/blob/582e368b6867ed8f4e7683e7bd3ffe52f964ef53/src/index.html#L12

Reply:
Great suggestion. I will look into it.